app/controllers/home_controller.rb

```diff
@@ -22,16 +23,41 @@ def set_host_info
 
   def set_all_markers
     @all_markers = []
-    Host.where.not(latitude: nil, longitude: nil).each do |h|
+    lat = [32.7490..34.7490]
+    lon = [-85.3880..-83.3880]
+    if @current_location.present?
+      new_lat = @current_location.data["lat"].to_f
+      new_lon = @current_location.data["lon"].to_f
+      lat = [(new_lat - 1)..(new_lat + 1)]
+      lon = [(new_lon - 1)..(new_lon + 1)]
+    end
+    Host.where(latitude: lat, longitude: lon).each do |h|
       @all_markers.push(
         {
           "lat": h.latitude,
           "lng": h.longitude,
           "infowindow": "<div>#{h.name}<br><b>Currently In Line: #{h.line_count_today}</b><br><a target='_blank' href='#{h.google_maps_url}'>Route me here!</a></div>",
-          "icon": "http://chart.apis.google.com/chart?chst=d_map_pin_letter&chld=1|FE6256|000000"
         }
       )
     end
     @all_markers = @all_markers.to_json.html_safe
   end
+
+  def set_current_location
+    @map_center = {
+      lat: 33.7490,
+      lng: -84.3880
+    }.to_json.html_safe
```

Comment:
What do these "magic numbers" represent? 
May be good to put them in an a constant / variable with a descriptive name.

Also, if they are GA specific, maybe would make sense to have them as an env or config variable. So if people are rolling out in different states this is not hard-coded into the app

Reply:
yep good idea, that's ATL lat lon, ill make them env vars